backend/index.js

```diff
@@ -0,0 +1,43 @@
+import express from "express";
+import session from "express-session";
+import fs from "fs";
+
+const app = express();
+
+let counter;
+
+fs.readFile("temp/my-counter-visited-times.txt", "utf8", (err, data) => {
+  if (err) {
+    console.error(err);
+    return;
+  } else {
+    counter = Number(data);
+  }
+});
```

Comment:
Read file is async, it does not give you data instantly please use readFileSync instead.
Also please use the returned value instead of callback functions 
e.g.
```
try {
  const data = readFileSync(path, 'utf8');
} catch (e) {
  console.log('ohno');
  <terminate the code here, return won't work as we are not inside of a function>
}
```